src/apispec/core.py

```diff
@@ -47,6 +49,21 @@ def to_dict(self):
             if v != {}
         }
 
+    def example(self, name, component=None, **kwargs):
+        """Add a new example to the spec
+
+        :param str name: identifier by which example may be referenced.
+        :param dict component: schema definition.
```

Comment:
Uh-oh, looks like we let this slip through.

Should be "example fields".

Reply:
So, it should be "Add example fields to the spec", right?

Reply:
or maybe: "Add an example which can be referenced", just like we have it in `parameter`'s docstring?

Reply:
Yeah. Consistency first.

Reply:
I've added the changes. Thanks for the reviews